<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>databaseLoginDialog</class>
 <widget class="QDialog" name="databaseLoginDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Database Login - Schedule Keeper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="address_label">
     <property name="minimumSize">
      <size>
       <width>388</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="address_input">
     <item>
      <widget class="QLineEdit" name="address_lineedit">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>31</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>192.168.1.1, localhost, etc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="remember_address_checkbox">
       <property name="text">
        <string>Remember</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="username_label">
     <property name="minimumSize">
      <size>
       <width>388</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="username_input">
     <item>
      <widget class="QLineEdit" name="username_lineedit">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="remember_username_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remember</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="password_label">
     <property name="minimumSize">
      <size>
       <width>388</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="password_input">
     <item>
      <widget class="QLineEdit" name="password_lineedit">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>31</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="remember_password_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remember</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="login_pushbutton">
     <property name="minimumSize">
      <size>
       <width>388</width>
       <height>33</height>
      </size>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
